app/models/authentication_event.rb

```diff
@@ -0,0 +1,17 @@
+class AuthenticationEvent < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id'
+
+  validates :email_submitted, presence: true
+  validates :event_type, presence: true
+  validates :user_id, presence: true
+  
+  # Logs an email validation event
```

Comment:
The `log_email_validation` method should be a class method (self.log_email_validation) to avoid the need to instantiate an object just to log an event.